packages/pigeon/platform_tests/test_plugin/android/src/test/kotlin/com/example/test_plugin/AllDatatypesTest.kt

```diff
@@ -46,33 +46,52 @@ internal class AllDatatypesTest {
   }
 
   @Test
-  fun testHasValues() {
+  fun testRoundtripNullValues() {
+    val everything = AllNullableTypes()
+    val codec = FlutterIntegrationCoreApi.codec
+    val encoded = codec.encodeMessage(everything)
+    encoded?.rewind()
+    val decoded = codec.decodeMessage(encoded)
+    assertEquals(everything, decoded)
+  }
+
+  fun getHasValues(): AllNullableTypes {
```

Comment:
How about something like "getFullyPopulatedAllNullableTypes"? I had no idea from the name what this was going to return.